app/src/main/java/com/macbitsgoa/events/aboutfest/OrganisersAdapter.java

```diff
@@ -0,0 +1,111 @@
+package com.macbitsgoa.events.aboutfest;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+
+import com.facebook.drawee.view.SimpleDraweeView;
+import com.macbitsgoa.events.R;
+
+import java.util.ArrayList;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+public class OrganisersAdapter extends RecyclerView.Adapter<OrganisersAdapter.ViewHolder> {
```

Comment:
break the line before `extends`. (Else we need to scroll horizontally to see full line)